Client/Teztez/Core/Extensions/UIView/UIView+GradientLayer.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  UIView+GradientLayer.swift
+//  Teztez
+//
+//  Created by Almas Zainoldin on 4/25/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    enum Direction {
+        case topToBottom
+        case bottomToTop
+        case leftToRight
+        case rightToLeft
+    }
+
+    func applyGradient(colors: [Any]?, locations: [NSNumber]? = [0.0, 1.0], direction: Direction = .leftToRight) {
```

Comment:
`CAGradientLayer` obviously accepts `[Any]` array as a value for `colors` property.
But can we set `func applyGradient(colors: [UIColor], ...)` and pass it inside? (It will downcast to [Any])